File: Assets/VirtualCity/ProtoDefine/Java/building/ReqRemoveHousePartMessage.java
package com.kingston.jforgame.server.game.building.message;

import com.baidu.bjf.remoting.protobuf.FieldType;
import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;
import com.kingston.jforgame.server.game.Modules;
import com.kingston.jforgame.server.game.building.BuildingDataPool;
import com.kingston.jforgame.server.game.building.entity.PutStatus;
import com.kingston.jforgame.socket.annotation.MessageMeta;
import com.kingston.jforgame.socket.message.Message;

import java.util.List;

@MessageMeta(module = Modules.BULIDING,cmd = BuildingDataPool.REQ_REMOVEHOUSEPART)
public class ReqRemoveHousePartMessage extends Message {
    @Protobuf(order = 1)
    private String houseId;
    @Protobuf(order = 2,fieldType = FieldType.STRING)
    private List<String> putStatusId;

    public ReqRemoveHousePartMessage() {
    }

    public ReqRemoveHousePartMessage(String houseId, List<String> putStatusId) {
        this.houseId = houseId;
        this.putStatusId = putStatusId;
    }

    public List<String> getPutStatusId() {
        return putStatusId;
    }

    public void setPutStatusId(List<String> putStatusId) {
        this.putStatusId = putStatusId;
    }

    public String getHouseId() {
        return houseId;
    }

    public void setHouseId(String houseId) {
        this.houseId = houseId;
    }


}

File: Assets/VirtualCity/ProtoDefine/ReqRemoveHousePartMessage.cs
using System.Collections.Generic;
using ProtoBuf;
namespace ProtoDefine {
[ProtoContract]



public class ReqRemoveHousePartMessage {
[ProtoMember(1)]
    public string houseId;
[ProtoMember(2)]
    public List<string> putStatusId;

    public ReqRemoveHousePartMessage() {
    }

    public ReqRemoveHousePartMessage(string houseId, List<string> putStatusId) {
        this.houseId = houseId;
        this.putStatusId = putStatusId;
    }

    public List<string> getPutStatusId() {
        return putStatusId;
    }

    public void setPutStatusId(List<string> putStatusId) {
        this.putStatusId = putStatusId;
    }

    public string getHouseId() {
        return houseId;
    }

    public void setHouseId(string houseId) {
        this.houseId = houseId;
    }


}
}